src/utils/chain-ops.js

```diff
@@ -0,0 +1,137 @@
+// Utilities for doing basic operations with chain.
+
+import { Keyring } from '@polkadot/keyring';
+import { randomAsHex, cryptoWaitReady } from '@polkadot/util-crypto';
+import { u8aToHex, formatBalance } from '@polkadot/util';
+import { isHexWithGivenByteSize, asDockAddress } from './codec';
+
+// XXX: Following info can be fetched from chain. Integrating in DockAPI object is an options.
+const TESTNET_ADDR_PREFIX = 21;
+const MAINNET_ADDR_PREFIX = 22;
+const DECIMAL = 6;
+const SYMBOL = 'DCK';
+
+// The methods below intentionally take an `ApiPromise` object to decouple from DockAPI.
+
+// Generate an account. If `secretUri` is not passed, a random one is generated
+export async function generateAccount({ secretUri, type = 'sr25519', network = 'test' }) {
+  if (['ed25519', 'sr25519', 'ecdsa'].indexOf(type) === -1) {
+    throw new Error(`${type} is not a valid type`);
+  }
+  if (network !== 'test' && network !== 'main') {
+    throw new Error('Network must be "test" or "main" or "dev"');
```

Comment:
This error still gets thrown when `network === "dev"`

Reply:
Oh, will fix.